contracts/Tokenomics.sol

```diff
@@ -853,11 +865,18 @@ contract Tokenomics is TokenomicsConstants {
         // The topUpUnitFraction was checked before and if it were zero, pendingRelativeTopUp would be zero as well
         if (totalIncentives > 0) {
             // Summation of all the unit top-ups and total amount of top-ups per epoch
+            // TODO: update the formula
             // topUp = (pendingRelativeTopUp * totalTopUpsOLAS * topUpUnitFraction) / (100 * sumUnitTopUpsOLAS)
-            totalIncentives *= mapEpochTokenomics[epochNum].epochPoint.totalTopUpsOLAS;
-            totalIncentives *= mapEpochTokenomics[epochNum].unitPoints[unitType].topUpUnitFraction;
-            uint256 sumUnitIncentives = uint256(mapEpochTokenomics[epochNum].unitPoints[unitType].sumUnitTopUpsOLAS) * 100;
-            totalIncentives = mapUnitIncentives[unitType][unitId].topUp + totalIncentives / sumUnitIncentives;
+            if (mapDonationPoints[epochNum].moreDonationPower) {
```

Comment:
The finalization of top-ups is based on formulas from the doc, and here they are not exactly correct. They are corrected in a next PR.